pages/home.html

```diff
@@ -2,9 +2,47 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="../css/bootstrap.css">
+    <link rel="stylesheet" href="../css/master.css">
+    <script src="https://kit.fontawesome.com/6854d7f83d.js" crossorigin="anonymous"></script>
     <title>Home Page</title>
   </head>
-  <body>
+
+  <main id=homeBackground>
+    <body >
+
+    <div class="container">
+      <h1 id=homeHeading>Live well in Dundee</h1>
+    </div>
+
+<div class="card" style="width:500px">
+    <img class="card-img-top" src="../pictures/little_green_larder.jpg" alt="Little green larder image">
+    <div class="card-body">
+    <h4 class="card-title">The Little Green Larder</h4>
+    <p class="card-text">The Little Green Larder offers a plastic free shopping experience, with zero waste alternatives to inspire us to create a greener future.</p>
+    <a href="#" class="btn btn-primary stretched-link">Click to visit page</a>
+  </div>
+</div>
+
+<div class="card" id=greenCard style="width:500px">
+   <img class="card-img-top" src="../pictures/green_living.jpg" alt="Little green larder image">
+   <div class="card-body">
+   <h4 class="card-title">Green Living</h4>
+   <p class="card-text">Learn about the impact we are having on out enviroment as well as recipe tips and tricks to help you live better.</p>
+   <a href="#" class="btn btn-primary stretched-link">Click to visit page</a>
+  </div>
+</div>
 
   </body>
+
+  <footer>
+    <div class="footer">
+      <div class="icons">
+        <a href="https://www.facebook.com/thelittlegreenlarder/" target="_blank"><em class="fab fa-facebook-square fa-2x"></em></a>
+        <a href="https://www.instagram.com/thelittlegreenlarder/" target="_blank"><em class="fab fa-instagram fa-2x"></em></a>
+     </div>
+    </div>
+  </footer>
+
+</main>
```

Comment:
You should end the `<main>` tag before you end the `<body>` tag.
Keep the footer within the `<body>` tag as well

Reply:
ok, done